internal/config/node/node.go

```diff
@@ -55,6 +55,13 @@ func ValidateConfig() error {
 			activated: &systemdHasCollectMode,
 			fatal:     false,
 		},
+		{
+			name:      "fs.may_detach_mounts sysctl",
+			init:      checkFsMayDetachMounts,
+			err:       &checkFsMayDetachMountsErr,
+			activated: nil,
+			fatal:     true,
```

Comment:
The only question is whether to make it fatal. I'm in favor to do so, and this is gross misconfiguration which will result in various failures later, so it's better to bail out.

Reply:
Basically if we fail here it means RPM packaging is broken or something like that.